Application/DefaultSettings.json

```diff
@@ -728,6 +728,126 @@
         }
       ]
     },
+    "Game": "T7",
+      "Maps": [
```

Comment:
invalid json here

Reply:
is it still wrong? im just now seeing this